detections/application/splunk_risky_command_abuse_ml_detection.yml

```diff
@@ -0,0 +1,67 @@
+name: Splunk Risky Command Abuse ML Detection
+id: 19d0146c-2eae-4e53-8d39-1198a78fa9ca
+version: 1
+date: '2022-05-27'
+author: Abhinav Mishra, Kumar Sharad and Xiao Lin, Splunk
+type: Anomaly
+datamodel:
+- Splunk_Audit
+description: 'This detection utilize machine learning model named "risky_command_abuse" trained from "Splunk 
+  Risky Command Abuse ML Model Baseline". It should be scheduled to run every hour 
+  to detect whether a user has run searches containing risky keywords with abnormal long running time in the 
```

Comment:
rename "to detect whether a user has run searches containing risky keywords with abnormal long running time" 

to 

"to detect whether a user has ran searches containing risky SPL from this list https://docs.splunk.com/Documentation/Splunk/latest/Security/SPLsafeguards#Commands_that_trigger_the_warninga with abnormally long running time"

Reply:
done